contrib/hack/setup-kubectl.sh

```diff
@@ -48,6 +48,6 @@ D_HOST=${D_HOST%:*}    # remove trailing port #
 # Setup our credentials
 NO_TTY=1 kubectl config set-credentials service-catalog-creds --username=admin --password=admin
 #NO_TTY=1 kubectl config set-cluster service-catalog-cluster --server=https://${D_HOST}:${PORT} --certificate-authority=/var/run/kubernetes-service-catalog/apiserver.crt
-NO_TTY=1 kubectl config set-cluster service-catalog-cluster --server=http://${D_HOST}:${PORT}
+NO_TTY=1 kubectl config set-cluster service-catalog-cluster --server=https://${D_HOST}:${PORT}
```

Comment:
It's set later and has been set for a long time, it was just being overridden.

See 
https://github.com/kubernetes-incubator/service-catalog/pull/991/files#diff-6558d4846dfc9ea670e39f25e102e98fR41 
for the existing use of the CA

and 
https://github.com/kubernetes-incubator/service-catalog/pull/991/files#diff-6c378e3d38ea115bb705ee0d2e9967f3L39
for the deleted hidden override